python/artm/batches_utils.py

```diff
@@ -0,0 +1,96 @@
+import os
+import glob
+
+from . import wrapper
+from wrapper import constants
+from wrapper import messages_pb2 as messages
+
+DICTIONARY_NAME = 'dictionary'
+
+
+class Batch(object):
+    def __init__(self, file_name):
+        self._file_name = file_name
```

Comment:
`file_name` —> `filename`

A list of my arguments: [[1]](http://english.stackexchange.com/questions/5366/which-is-correct-filename-file-name-or-filename), [[2]](https://en.wikipedia.org/wiki/Filename), [[3]](https://docs.python.org/2/library/os.path.html) (they're using "filename" in the text)